spec/factories/category_factory.rb

```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+  factory :category do
+    name 'First category'
```

Comment:
When someone will need to use this factory and your data will be not acceptable he will need to change the factory. This action will make your test red and you college will spend some time to fix your spec. Factory should contain random data, if you need to apply custom test data use, for example: `let(:category){ FactoryGirl.build(:category, name: 'First category') }`. 
On other hand FactoryGirl is very useful to insert valid test data to development DB, so that if you apply FactoryGirl attributes with constant data after second FactoryGirl exception will be thrown.